h2/config.py

```diff
@@ -87,13 +93,18 @@ def __init__(self,
                  header_encoding='utf-8',
                  validate_outbound_headers=True,
                  normalize_outbound_headers=True,
-                 validate_inbound_headers=True):
+                 validate_inbound_headers=True,
+                 logger=None):
         self.client_side = client_side
         self.header_encoding = header_encoding
         self.validate_outbound_headers = validate_outbound_headers
         self.normalize_outbound_headers = normalize_outbound_headers
         self.validate_inbound_headers = validate_inbound_headers
 
+        default_logger = logging.getLogger(__name__)
```

Comment:
So, I think we don't want to do this if we can avoid it: we'd rather that there be no default logger, because that's an easy way to accidentally add logging to hyper-h2. Instead, we'd like the logger to be injected in.

This may mean we need a dummy logger that isn't part of the logging system that we can use as our "default" logger.